lib/ardb/migration.rb

```diff
@@ -4,16 +4,18 @@ module Ardb
 
   class Migration
 
-    attr_reader :identifier, :class_name, :file_name, :file_path
-    attr_reader :source
+    attr_reader :identifier, :migrations_path
+    attr_reader :class_name, :file_name, :file_path, :source
 
-    def initialize(identifier)
+    def initialize(identifier, ardb_config)
```

Comment:
@kellyredding - I reorderd them. I didn't think about it when I was adding it but I agree, it feels more appropriate. Thanks.